### DISCUSSION TITLE
How to see Lora trigger words during use?

### DISCUSSION THREAD

**Discussion by jj05690569** [runew0lf/RuinedFooocus#97] [Q&A]
Hi, I created the same named .txt file as the lora file name, which contains all trigger words. During use, I can only see all trigger words listed under the "Info" tab after a generation is completed; is there a way to see which trigger words are available before generation? Thanks

**Comment by runew0lf** [comment]
at the moment no, i added the function to just use the trigger word so i didnt have to remember it, some of them are really weird! :D

**Comment by runew0lf** [comment]
Okies the new update, it doesnt automatically add the trigger words now however it does display them in a trigger word box underneath the loras so you can copy and paste as you please

**Comment by jj05690569** [comment]
Thanks for replying! Ya I totally know what you mean haha

**Comment by runew0lf** [comment]
No worries man, you do get a quicker response in our discord btw :D

**Comment by jj05690569** [comment]
Thanks for the tip!
